mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -3672,9 +3677,6 @@ msgstr "I used to enjoy coming here every weekend to look at the paintings. \n N
 msgid "spyder_cottonart_florist"
 msgstr "What do you think of the blue painting of a ship in the harbour? \n It's in the impressionist style - that means it tries to capture how light looks and behaves. \n See the small red sun reflected along the waves of the water?"
 
-msgid "spyder_cottonart_intro"
```

Comment:
This is unused?

Reply:
Yes, identical to **spyder_cottonart_shopkeeper** and unused.